File: src/view/history_view.py
from src.model.history import History
from src.view.bar_view import bar


def print_history(history_list: list[History]):
    bar("History")
    if len(history_list) == 0:
        print("Empty history")
        return
    for i in range(len(history_list)):
        print(f"{i + 1}. {history_list[i].timestamp} - operation: {history_list[i].operation} - sold: {history_list[i].wallet.sold}")

File: lib/view/history_view.rb
require_relative '../model/history'
require_relative 'bar_view'

def print_history(history_list)
  bar("History")
  if history_list.empty?
    puts "Empty history"
    return
  end
  history_list.each_with_index do |history, index|
    puts "#{index + 1}. #{history.timestamp} - operation: #{history.operation} - sold: #{history.wallet.sold}"
  end
end
